src/bootstrap-table.js

```diff
@@ -2532,9 +2532,15 @@ class BootstrapTable {
       }
 
       if (params.hasOwnProperty('replace') && params.replace) {
-        this.options.data[params.index] = params.row
+        const oldRow = this.data[params.index]
+
+        Object.keys(oldRow).forEach(function (key) {
+          oldRow[key] = null
+        })
+
+        $.extend(this.data[params.index], $.extend(oldRow, params.row))
```

Comment:
What happens if you do:

```js
this.data[params.index] = params.row
```

I think the logic is too complicated

Reply:
That was my first commit for the replacing part.
But as i already wrote in the post above, if i reassigne the value (`= params.row`) it don't work.
If i use the too complicated ugly fix, it works fine locally but not on the editor.

I'm quite confused why it only works with `$.extend`  and why the complicated fix don't work on the editor.